[Feature] Automatically take screenshot on failure/error (#392)
symfony/panther: README.md

While not strictly mandatory, this extension dramatically improves the testing experience by boosting the performance and
allowing to use the [interactive debugging mode](#interactive-mode).

> When using the extension in conjunction with `PANTHER_SCREENSHOT_DIR` environment variable, test using the Panther

Reviewer: ```suggestion
When using the extension in conjunction with the `PANTHER_SCREENSHOT_DIR` environment variable, tests using the Panther
```

> client that fail or error (after the client is created) will get automatically get a screenshot taken to help

Reviewer: ```suggestion
client that fail or error (after the client is created) will automatically get a screenshot taken to help
```

> debug.

Reviewer: ```suggestion
debugging.
```

* `PANTHER_WEB_SERVER_ROUTER`:  to use a web server router script which is run at the start of each HTTP request
* `PANTHER_EXTERNAL_BASE_URI`: to use an external web server (the PHP built-in web server will not be started)
* `PANTHER_APP_ENV`: to override the `APP_ENV` variable passed to the web server running the PHP app

> * `PANTHER_SCREENSHOT_DIR`: to set a base directory for your screenshots (ie `./screenshots`)

Reviewer: ```suggestion
* `PANTHER_SCREENSHOT_DIR`: to set a base directory for your screenshots (e.g. `./screenshots`)
```
